.changeset/funny-islands-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@linear/sdk": patch
```

Comment:
nit: I would call this a `minor` change since it carries some risk of affecting some downstream consumers